CMakeLists.txt

```diff
@@ -8,8 +11,10 @@ endif()
 
 include(CTest)
 
+set(CMAKE_C_STANDARD 99)
```

Comment:
We should consider uping this to 17 in a future PR

Reply:
20 /s